.github/workflows/verify-std-check.yml

```
@@ -39,6 +39,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: kani
+          fetch-depth: 2
```

My bad... Thanks for fixing it. We might need more than 2 here since a PR can have multiple commits.
```suggestion
          fetch-depth: 0
```